pyuvm/s17_pyuvm_reg_enumerations.py

```diff
@@ -4,16 +4,16 @@
 from enum import Enum
 
 
-PYUVM_REG_DATA_WIDTH = 64
-PYUVM_REG_ADDR_WIDTH = 64
+UVM_REG_DATA_WIDTH = 64
+UVM_REG_ADDR_WIDTH = 64
 
 # 17.2.1.1 uvm_reg_data_t
-pyuvm_reg_data_t = int
+uvm_reg_data_t = int
 
 # 17.2.1.3 uvm_reg_addr_t
-pyuvm_reg_addr_t = int
+uvm_reg_addr_t = int
 
-pyuvm_reg_policy_t = [
+uvm_reg_policy_t = [
     "RO",       # no effect, R: no effect.
```

Comment:
changing `pyuvm_` into `uvm` to be consistent across the project. Shall we define the standard type as `cocotb::BinaryValue or LogicaArray` ?